docs/tests/getting-started/github-actions.yml

```diff
@@ -23,9 +23,16 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v2
 
+      # You need to run `dagger project init` locally before and commit the cue.mod directory to the repository with its contents
+      - name: Project update
+        uses: dagger/dagger-for-github@v2
+        with:
+          version: 0.2
+          # To pin external dependencies, you can use `project update github.com/[package-source]@v[n]`
+          args: project update
+
       - name: Deploy to Netlify
         uses: dagger/dagger-for-github@v2
         # See all options at https://github.com/dagger/dagger-for-github
         with:
-          workdir: pkg/universe.dagger.io/examples/todoapp
```

Comment:
Why that line has been removed?

Reply:
Removed it on the assumption that it would be a generic GitHub Action for the "getting started" section of the documentation. On the actual [directory](https://github.com/dagger/dagger/tree/main/pkg/universe.dagger.io/examples/todoapp) in the official example repository, the GitHub action would not work due to `cue.mod` not being included in the repository anyway.